tests/entropy_multicomponent_12_edge_case_3_components/screen-output

```diff
@@ -165,9 +165,9 @@ Number of degrees of freedom: 85 (18+4+9+9+9+9+9+9+9)
    Solving entropy_comp3 system ... 0 iterations.
    Copying properties into prescribed temperature field... done.
    Copying properties into prescribed compositional field density_field... done.
-   Solving Stokes system (GMG)... 15+0 iterations.
-      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 3.28443e-17, 3.08336e-16, 1.15165e-16, 1.9666e-16, 1.40508e-16, 3.53323e-16, 0, 21.9867
-      Relative nonlinear residual (total system) after nonlinear iteration 10: 21.9867
+   Solving Stokes system (GMG)... 8+0 iterations.
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 3.28443e-17, 3.25245e-16, 1.37488e-16, 2.06377e-16, 2.67266e-16, 3.45618e-16, 0, 148.389
+      Relative nonlinear residual (total system) after nonlinear iteration 10: 148.389
```

Comment:
The entropy method seems to use less linear Stokes iterations now, but it looks as if the nonlinear solver converges more slowly as a result. I will try to run one of those tests until it converges and compare 9.5 and 9.6 to make sure both eventually converge.

I reviewed test changes until file 277/1043.